mysql-test/suite/galera/t/MDEV-30418.test

```diff
@@ -0,0 +1,24 @@
+--source include/galera_cluster.inc
+
+# Save original auto_increment_offset values.
+--let $node_1=node_1
+--let $node_2=node_2
+--source ../galera/include/auto_increment_offset_save.inc
+
+--connection node_2
+select @@wsrep_slave_threads;
+SET @cluster_address_orig = @@wsrep_cluster_address;
+SET GLOBAL wsrep_cluster_address=AUTO;
```

Comment:
If the purpose of changing `wsrep_cluster_address` is to take the node out of the cluster, `SET GLOBAL wsrep_cluster_address=''` might be better as `AUTO` is not considered a valid address string by Galera.

Reply:
Bug report https://jira.mariadb.org/browse/MDEV-30418 specifically states `wsrep_cluster_address=AUTO`, so this is ok.